lib/galaxy/tool_util/xsd/galaxy.xsd

```diff
@@ -5581,14 +5582,62 @@ A tool can have any number of EDAM operation references.
 ]]></xs:documentation>
     </xs:annotation>
     <xs:sequence>
-        <xs:element name="edam_operation" minOccurs="0" maxOccurs="unbounded">
-          <xs:simpleType>
-            <xs:restriction base="xs:string">
-              <xs:pattern value="operation_[0-9]{4}"></xs:pattern>
-            </xs:restriction>
-          </xs:simpleType>
-        </xs:element>
-      </xs:sequence>
-    </xs:complexType>
+      <xs:element name="edam_operation" minOccurs="0" maxOccurs="unbounded">
+        <xs:simpleType>
+          <xs:restriction base="xs:string">
+            <xs:pattern value="operation_[0-9]{4}"></xs:pattern>
+          </xs:restriction>
+        </xs:simpleType>
+      </xs:element>
+    </xs:sequence>
+  </xs:complexType>
+  <xs:complexType name="xrefs">
+    <xs:annotation>
+      <xs:documentation xml:lang="en"><![CDATA[
+Container tag set for the ``<xref>`` tags.
+A tool can refer multiple reference IDs.
+	
+```xml
+   <!-- Example: this tool is seqtk -->
+   <xrefs>      
+	<xref type="bio.tools">seqtk</xref>
+   </xrefs>
+   <!-- https://bio.tools/seqtk -->
+```
+   
+]]></xs:documentation>
+    </xs:annotation>
+      <xs:sequence>
+        <xs:element name="xref" type="xref" minOccurs="0" maxOccurs="unbounded">
+      </xs:element>
+    </xs:sequence>
+  </xs:complexType>
+  <xs:complexType name="xref">
+    <xs:annotation>
+      <xs:documentation xml:lang="en">Each xrefs element can contain one or
+more ``xref`` tag elements - each of which specifies reference information 
+according to a catalog or an other.
```

Comment:
I'd simplify the text to: `The ``xref`` element specifies reference information 
according to a catalog.`

Reply:
You're right, I had copied the 'citation' documentation : corrected in commit 17807f4